topics/climate/tutorials/pangeo-notebook/tutorial.md

```diff
@@ -0,0 +1,723 @@
+
+There are many ways to customize your plots and here we only give you what we think is important for creating publication ready figures:
+- Define the size of the figure
+- Choose to project data on a different projection.
+- Add coastline
+- Set the min and max values for plotting
+- Add a title, change colorbar title
+- Save figure into png
+
+ ```python
+fig = plt.figure(1, figsize=[15,10])
+ 
+# We're using cartopy to project our data.
+# (see documentation on cartopy)
+ax = plt.subplot(1, 1, 1, projection=ccrs.Mercator())
+ax.coastlines(resolution='10m')
+  
+# We need to project our data to the new projection and for this we use `transform`.
+# we set the original data projection in transform (here PlateCarree)
+dset.sel(time=(np.timedelta64(2,'D') + np.timedelta64(12,'h')))['pm2p5_conc'].plot(ax=ax, 
+                                                                                    transform=ccrs.PlateCarree(),
+                                                                                    vmin = 0, vmax = 35,
+                                                                                    cmap=cmc.roma_r)
+# One way to customize your title
+plt.title("Copernicus Atmosphere Monitoring Service PM2.5, 2 day forecasts\n 24th December 2021 at 12:00 UTC", fontsize=18)
+plt.savefig("CAMS-PM2_5-fc-20211224.png")
+```
+>    {: .code-in}
+
+And you should get the following plot:
+
+![Customized plot for CAMS PM2.5 December, 24th 2021 at 12:00 UTC](../../images/CAMS-PM2_5-fc-20211224.png)
+
+### Multi-plots
+
+- Here we would like to plot several times on the same figure in different sub-plots
+- We will not plots all the times (too many) but the first 24 forecasted values.
+
+1. Make a list of times and convert to pandas datetime to make it easier to format times when plotting
+ 
+```python
+list_times = np.datetime64('2021-12-22') + dset.time.sel(time=slice(np.timedelta64(0),np.timedelta64(1,'D')))
+print(pd.to_datetime(list_times).strftime("%d %b %H:%S UTC"))
+```
+
+2. We use the same plotting method as earlier, but we pass additional parameters:
+        - `vmin = 0`and `vmax = 35` to set the minimum and maximum values when plotting (this is useful to highlight features in your plot);
+        - `subplot_kws={"projection": proj_plot}` to project data on a non-default projection. See [cartopy projection](https://scitools.org.uk/cartopy/docs/v0.15/crs/projections.html) for more information about projections;
+        - `col='time'` because we will plot several `time`;
+        -  `col_wrap=4` to have a maximum of 4 plots per row. If we have more times to plot, then the next figures will be on another row;
+        - `robust=True` and `aspect=dset.dims["longitude"] / dset.dims["latitude"]` are additional parameters to make each subplot with a "sensible" figsize;
+        - `cmap=cmc.roma_r` to select a non-default and color-blind friendly colormap (see [scientific colormaps](https://www.fabiocrameri.ch/colourmaps/)).
```

Comment:
```suggestion
        
- `vmin = 0`and `vmax = 35` to set the minimum and maximum values when plotting (this is useful to highlight features in your plot)
- `subplot_kws={"projection": proj_plot}` to project data on a non-default projection. See [cartopy projection](https://scitools.org.uk/cartopy/docs/v0.15/crs/projections.html) for more information about projections.
- `col='time'` because we will plot several `time`;
-  `col_wrap=4` to have a maximum of 4 plots per row. If we have more times to plot, then the next figures will be on another row.
- `robust=True` and `aspect=dset.dims["longitude"] / dset.dims["latitude"]` are additional parameters to make each subplot with a "sensible" figsize.
- `cmap=cmc.roma_r` to select a non-default and color-blind friendly colormap (see [scientific colormaps](https://www.fabiocrameri.ch/colourmaps/)).
```

Reply:
This bullet point list doesn't render correctly; I think that it will fix it.